iml-manager-env/src/lib.rs

```diff
@@ -247,16 +255,12 @@ pub fn get_use_stratagem() -> bool {
     string_to_bool(get_var("USE_STRATAGEM"))
 }
 
-pub fn get_action_runner_connect() -> String {
```

Comment:
nit: this was probably the cleaner approach

Reply:
I need to use different  URI parsers (`hyper` and `hyperlocal`) in `iml-action-client`, so these needed to be separated.